### PR TITLE
Run CS sniffer over tests.

### PR DESCRIPTION
**Pull request by dereuromark**
Run automated sniffer part over tests dir

### CODE REVIEW

File: tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectWithInterfaceTest.php [hunk 1]
```diff
@@ -27,11 +28,14 @@ public function setUp(): void
...
-      public function testClassHasInterface()
-      {
-            $this->assertInstanceOf('Foo\MyInterface', new \Foo\MyClassWithInterface());
-      }
+    /**
+     * @return void
+     */
+    public function testClassHasInterface()
+    {
+          $this->assertInstanceOf('Foo\MyInterface', new \Foo\MyClassWithInterface());
```
Comment (gechetspr) [inline tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectWithInterfaceTest.php:39]: Can we put this to the use section?

File: tests/Propel/Tests/Generator/Builder/Om/QueryBuilderInheritanceTest.php [hunk 2]
```diff
@@ -137,25 +152,42 @@ public function testFindPkSimpleWithSingleTableInheritanceReturnCorrectClass()
...
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreEmployee', BookstoreEmployeeQuery::create()->findPk($employee->getId()),
-            'findPk() return right object : BookstoreEmployee');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreManager', BookstoreEmployeeQuery::create()->findPk($manager->getId()),
-            'findPk() return right object : BookstoreManager');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreCashier', BookstoreEmployeeQuery::create()->findPk($cashier1->getId()),
-            'findPk() return right object : BookstoreCashier');
-        $this->assertInstanceOf('\Propel\Tests\Bookstore\BookstoreCashier', BookstoreEmployeeQuery::create()->findPk($cashier2->getId()),
-            'findPk() return right object : BookstoreCashier');
+        $this->assertInstanceOf(
+            '\Propel\Tests\Bookstore\BookstoreEmployee',
```
Comment (gechetspr) [inline tests/Propel/Tests/Generator/Builder/Om/QueryBuilderInheritanceTest.php:156]: can we use BookstoreEmployee::class here and in in other places in this method?

File: tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectWithInterfaceTest.php (hunk 1 above)
Comment (dereuromark) [reply tests/Propel/Tests/Generator/Builder/Om/GeneratedObjectWithInterfaceTest.php:39]: Next sniffer run, there are more to come.

File: tests/Propel/Tests/Generator/Builder/Om/QueryBuilderInheritanceTest.php (hunk 2 above)
Comment (dereuromark) [reply tests/Propel/Tests/Generator/Builder/Om/QueryBuilderInheritanceTest.php:156]: Next sniffer run, there are more to come (with manual ones).

**Pull request merged**